app/scripts/controllers/info.ctrl.js

```diff
@@ -1,8 +1,36 @@
 'use strict';
 
-angular.module('yourStlCourts').controller('InfoCtrl', function ($anchorScroll) {
+angular.module('yourStlCourts').controller('InfoCtrl', function ($anchorScroll,courts, $state,municipalities) {
   var ctrl = this;
+  ctrl.citySearchGroups = [];
+  ctrl.municipalities = municipalities;
+  ctrl.selectedCitySearchGroup = null;
   ctrl.scrollTo = function(id){
     $anchorScroll(id);
   }
+
+  ctrl.onCitySearchGroupSelected  = function(){
```

Comment:
This function doesn't do anything

Reply:
This controller should have been deleted